pkg/deploytest/testreplica.go

```diff
@@ -43,7 +41,7 @@ type TestReplica struct {
 	Membership []t.NodeID
 
 	// Network transport subsystem.
-	Net modules.ActiveModule
+	NetLink Link
```

Comment:
`NetLink Link` says nothing about network transport

Reply:
Without comment it wouldn't be easy to understand that

Reply:
Changed to `Transport net.Transport`